patches/build-config-compiler-BUILD.gn.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
+index de1cd6efc67b4c93ffd73896f479f3839cb37da5..d71860fce442ff958ee1b16f832c40b094283b83 100644
+--- a/build/config/compiler/BUILD.gn
++++ b/build/config/compiler/BUILD.gn
+@@ -594,7 +594,6 @@ config("compiler") {
+   }
+ 
+   if (is_clang && !is_nacl && current_os != "zos") {
+-    cflags += [ "-fcrash-diagnostics-dir=" + clang_diagnostic_dir ]
```

Comment:
is this going to be added back in prior to merge?

Reply:
cc: @cdesouza-chromium

Reply:
depends if this commit exists in `reclient` that's distributed in cr121.
https://github.com/bazelbuild/reclient/commit/58a8c079e9289654758d5cb161f131e73074cb0b